Makefile

```diff
@@ -57,4 +57,12 @@ check-within-openshift:
 	pytest-3 -k test_update
 
 requre-purge-files:
-	requre-patch purge --replaces "requests.sessions%send:Date:str:Fri, 01 Nov 2019 13-36-03 GMT" --replaces 'requests.sessions%send:ETag:str:W/"1e51b8e1c48787a433405211e9e0fe61"' --replaces "requests.sessions%send:elapsed:float:0.2" --replaces "metadata:latency:float:0" --replaces "requests.sessions%_content:expires_at:str:2019-11-01T14:35:53Z" --replaces "requests.sessions%_content:token:str:v1.1bd88d399b8c70e8b88e22cbdaa72abad5e399db" --replaces ":X-GitHub-Request-Id:str:DE82:1411D:42114C4:4F8EF0E:5DBC34B9" --replaces ":X-RateLimit-Reset:str:1573052591" tests_requre/test_data/test_*/*.yaml
+	requre-patch purge --replaces "requests.sessions%send:Date:str:Fri, 01 Nov 2019 13-36-03 GMT" \
```

Comment:
What about using pre-commit hook so we have it in one place. (You can run `pre-commit run --all requre-purge`.)

Reply:
https://github.com/packit-service/requre/blob/master/.pre-commit-hooks.yaml

Reply:
Sure, but ideally in new PR. are you okay with this?

Reply:
Ah, now I understand, that we can remove this target, or better to say, use your mentioned command. makes sesnse

Reply:
Please, update the hook if you need some other replacements.

Reply:
nope, I hope that these are fine, but question is if we should also remove latency data, to not have differences there.

Reply:
I am ok with seeing the changes on latency.

Reply:
> I am ok with seeing the changes on latency.

so everything could be fine now, lets wait for seconod PR, and then I'll rebase it and we'll see :-)